samples/scala/demo/conf/securesocial.conf

```diff
@@ -269,6 +269,14 @@ securesocial {
 		scope = ""
 	}
 
+	chatwork {
+		authorizationUrl = "https://www.chatwork.com/packages/oauth2/login.php"
+		accessTokenUrl = "https://oauth.chatwork.com/token"
+		clientId = your_client_id
+		clientSecret = your_client_secret
+		scope = "rooms.all:read_write users.all:read"
```

Comment:
Even though this is a demo app, the scope that it requests should be minimal, and I think `users.profile.me:read` should be enough.

Reply:
I fixed in next commit.